src/loaders/api/api.js

```diff
@@ -198,7 +198,7 @@ export function setAPI (agentIdentifier, forceDrain, runSoftNavOverSpa = false)
       drain(agentIdentifier, 'api')
     }).catch(() => {
       warn('Downloading runtime APIs failed...')
-      drain(agentIdentifier, 'api', true)
+      instanceEE.aborted = true
```

Comment:
Just pointing out that we're aborting the entire agent because the async apis (finished, setErrorHandler, addToTrace, addRelease) failed to import. You sure?